server/adaptors/integrations/__data__/repository/k8s/data/sample.json

```diff
@@ -0,0 +1,4835 @@
+[
+  {
+    "kubernetes": {
+      "node": {
+        "_domain": {
+          "labels": {
+            "node": {
+              "kubernetes": {
+                "io/instance-type": "m5.2xlarge"
+              }
+            },
+            "kubernetes": {
+              "io/arch": "amd64",
+              "io/os": "linux",
+              "io/hostname": "ip-10-0-0-253.ec2.internal"
+            },
+            "topology": {
+              "kubernetes": {
+                "io/zone": "us-east-1a",
+                "io/region": "us-east-1"
+              },
+              "ebs": {
+                "csi": {
+                  "aws": {
+                    "com/zone": "us-east-1a"
+                  }
+                }
+              }
+            },
+            "k8s": {
+              "io/cloud-provider-aws": "22771173a877370a23796c9759ec12cf"
+            },
+            "eks": {
+              "amazonaws": {
+                "com/capacityType": "ON_DEMAND",
+                "com/nodegroup-image": "ami-06c9b6a12f5bd0a96",
+                "com/nodegroup": "playground-observability"
+              }
+            },
+            "failure-domain": {
+              "beta": {
+                "kubernetes": {
+                  "io/zone": "us-east-1a",
+                  "io/region": "us-east-1"
+                }
+              }
+            },
+            "beta": {
+              "kubernetes": {
+                "io/arch": "amd64",
+                "io/os": "linux",
+                "io/instance-type": "m5.2xlarge"
+              }
+            }
+          }
+        },
+        "memory": {
+          "allocatable": {
+            "bytes": 31792103424
+          },
+          "capacity": {
+            "bytes": 32833339392
+          }
+        },
+        "pod": {
+          "allocatable": {
+            "total": 58
+          },
+          "capacity": {
+            "total": 58
+          }
+        },
+        "name": "ip-10-0-0-253.ec2.internal",
+        "cpu": {
+          "allocatable": {
+            "cores": 7.91
+          },
+          "capacity": {
+            "cores": 8
+          }
+        },
+        "status": {
+          "ready": "true",
+          "unschedulable": false
+        }
+      }
+    },
+    "event": {
+      "duration": 32121077,
+      "domain": "kubernetes",
+      "dataset": "kubernetes.node"
```

Comment:
event.duration and event.dataset are also absent from the schema

Reply:
yes - need to extract the `event` object from the logs into an independent component

Reply:
will add these Fields here until the larger change will be applied